File: ntss-win/NKKWeight/Source/NKKWeightScaleDB/Models/Mst_wheel_chair.cs
namespace NKKWeightScaleDB.Models
{
    using Newtonsoft.Json;

    public class Mst_Wheel_Chair : BaseEntity
    {
        [JsonProperty(PropertyName = "wheelChairCd")]
        public string wheel_chair_cd { get; set; }

        [JsonProperty(PropertyName = "facilityCd")]
        public string facility_cd { get; set; }

        [JsonProperty(PropertyName = "fnWheelChairCd")]
        public string fn_wheel_chair_cd { get; set; }

        [JsonProperty(PropertyName = "wheelChairName")]
        public string wheel_chair_name { get; set; }

        [JsonProperty(PropertyName = "wheelChairWeight")]
        public string wheel_chair_weight { get; set; }

        [JsonProperty(PropertyName = "scaleDate")]
        public string scale_date { get; set; }

        [JsonProperty(PropertyName = "scaleUserId")]
        public string scale_user_id { get; set; }

        [JsonProperty(PropertyName = "isPersonal")]
        public string is_personal { get; set; }

        [JsonProperty(PropertyName = "patId")]
        public string pat_id { get; set; }

        [JsonProperty(PropertyName = "isDisp")]
        public string is_disp { get; set; }

        [JsonProperty(PropertyName = "isDel")]
        public string is_del { get; set; }

        [JsonProperty(PropertyName = "regDate")]
        public string reg_date { get; set; }

        [JsonProperty(PropertyName = "upDate")]
        public string up_date { get; set; }
    }
}

File: Source/ntss-src/ntss-core/src/main/java/jp/co/nikkiso/ntss/core/entity/MstWheelChair.java
package jp.co.nikkiso.ntss.core.entity;

import java.sql.Timestamp;

import org.seasar.doma.Entity;
import org.seasar.doma.GeneratedValue;
import org.seasar.doma.GenerationType;
import org.seasar.doma.Id;
import org.seasar.doma.Table;
import org.seasar.doma.jdbc.entity.NamingType;

import jp.co.nikkiso.ntss.core.entity.entityListener.BaseEntityListener;
import lombok.Getter;
import lombok.Setter;

/**
 * 車いすマスタのEntity.
 */
@Entity(listener = BaseEntityListener.class, naming = NamingType.SNAKE_LOWER_CASE)
@Table(name = "mst_wheel_chair")
@Getter
@Setter
public class MstWheelChair extends BaseEntity {

  /**
   * 車いす管理コード
   */
  @Id
  @GeneratedValue(strategy = GenerationType.IDENTITY)
  private Long wheelChairCd;
  /**
   * 施設コード
   */
  private String facilityCd;

  /**
   * FNW+で管理する施設内で一意な車いす管理コード
   */
  private String fnWheelChairCd;
  /**
   * 車いす名称
   */
  private String wheelChairName;

  /**
   * 車いす重量(g)
   */
  private Integer wheelChairWeight;
  /**
   * 車いす校正日
   */
  private Timestamp scaleDate;

  /**
   * 車いす校正者ID
   */
  private Long scaleUserId;
  /**
   * 個人所有フラグ
   */
  private String isPersonal;
  /**
   * 所有患者ID
   */
  private Long patId;
  /**
   * 表示フラグ
   */
  private String isDisp;
  /**
   *    削除フラグ
   */
  private String isDel;
}
